lxd/storage/drivers/utils.go

```diff
@@ -773,7 +773,9 @@ func BTRFSSubVolumeMakeRo(path string) error {
 	return err
 }
 
-// BTRFSSubVolumeMakeRw makes a sub volume read/write. Deprecated use btrfs.setSubvolumeReadonlyProperty().
+// BTRFSSubVolumeMakeRw makes a sub volume read/write.
+//
+// Deprecated: Use btrfs.setSubvolumeReadonlyProperty().
```

Comment:
Can we remove this function and update the callers to use btrfs.setSubvolumeReadonlyProperty